CONTRIBUTING.md

```diff
@@ -132,6 +133,7 @@ project's service account.
 - `GCLOUD_TESTS_GOLANG_FIRESTORE_KEY`: The path to the JSON key file of the
 Firestore project's service account.
 - `GCLOUD_TESTS_API_KEY`: API key for using the Translate API created above.
+- `GCLOUD_TESTS_GOLANG_BIGTABLE_PROJECT_ID`: Developers Console project's ID (e.g. doorway-cliff-677) for the Bigtable project. Bigtable project can be same as Firestore project or any project other than the general project.
```

Comment:
maybe we should signal in the variable name this is a secondary project id (something like `..._SECONDARY_BIGTABLE_PROJECT_ID` for copy backup testing and it is optional?

Reply:
Done